### PR TITLE
update opensora-hpcai jit_level

### PR DESCRIPTION
**Pull request by Songyuanwei**
# What does this PR do?

Fixes # (issue)

Adds # (feature)

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/mindspore-lab/mindone/blob/master/CONTRIBUTING.md)?
- [ ] Did you make sure to update the documentation with your changes? E.g. record bug fixes or new features in `What's New`. Here are the
      [documentation guidelines](https://github.com/mindspore-lab/mindcv/wiki/%E6%96%87%E6%A1%A3%E7%BC%96%E5%86%99%E8%A7%84%E8%8C%83)
- [ ] Did you build and run the code without any errors?
- [ ] Did you report the running environment (NPU type/MS version) and performance in the doc? (better record it for data loading, model inference, or training tasks)
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

@xxx

### CODE REVIEW

File: examples/opensora_hpcai/scripts/infer_t5.py [hunk 1]
```diff
@@ -38,7 +39,7 @@ def init_env(
     distributed: bool = False,
     max_device_memory: str = None,
     device_target: str = "Ascend",
-    enable_dvm: bool = False,
+    backend: str = "kbk",
```
Comment (CaitinZhao) [inline examples/opensora_hpcai/scripts/infer_t5.py:42]: directly to jit_level? not  display kbk, dvm, and ge.

**Review by SamitHuang — APPROVED** [review]

File: examples/opensora_hpcai/README.md [hunk 2]
```diff
@@ -143,9 +143,8 @@ Other useful documents and links are listed below.
 
 ## Installation
 
-1. Install MindSpore 2.3rc1 according to the [official instruction](https://www.mindspore.cn/install)
-> To use flash attention, it's recommended to use mindspore 2.3rc2 (release soon).
-
+1. Install MindSpore according to the [official instructions](https://www.mindspore.cn/install).
+    For Ascend devices, please install **CANN driver C18 (0517)** from [here](https://repo.mindspore.cn/ascend/ascend910/20240517/Ascend910B/) and install **MindSpore 2.3-master (0615)** from [here](https://repo.mindspore.cn/mindspore/mindspore/version/202406/20240615/master_20240615020018_43ccb91e45899b64fe31d304497ab17e3ada3cea_newest/unified/).
```
Comment (HaoyangLee) [inline examples/opensora_hpcai/README.md:147]: https://repo.mindspore.cn/ascend/ascend910/20240517/Ascend910B/
```suggestion
    For Ascend devices, please install **CANN driver C18 (0517)** from [here](https://repo.mindspore.cn/ascend/ascend910/20240517/) and install **MindSpore 2.3-master (0615)** from [here](https://repo.mindspore.cn/mindspore/mindspore/version/202406/20240615/master_20240615020018_43ccb91e45899b64fe31d304497ab17e3ada3cea_newest/unified/).
```
这一层目录里面只有driver和firmware包，而cann包在上一层目录。是否要写到上一层目录？

File: examples/opensora_hpcai/scripts/infer_t5.py [hunk 3]
```diff
@@ -10,6 +10,7 @@
 from tqdm import tqdm
 
 import mindspore as ms
+from mindspore._c_expression import ms_ctx_param
```
Comment (zhtmike) [inline examples/opensora_hpcai/scripts/infer_t5.py:13]: add a FIXME tag here to remove of using internal API

**Review by zhtmike — APPROVED** [review]

**Review by hadipash — APPROVED** [review]

File: examples/opensora_hpcai/scripts/infer_t5.py (hunk 1 above)
Comment (Songyuanwei) [reply examples/opensora_hpcai/scripts/infer_t5.py]: done

File: examples/opensora_hpcai/README.md (hunk 2 above)
Comment (Songyuanwei) [reply examples/opensora_hpcai/README.md]: done

File: examples/opensora_hpcai/scripts/infer_t5.py (hunk 3 above)
Comment (Songyuanwei) [reply examples/opensora_hpcai/scripts/infer_t5.py]: done

**Review by SamitHuang — APPROVED** [review]

File: examples/opensora_hpcai/README.md [hunk 4]
```diff
@@ -494,9 +493,18 @@ You may also see the example shell scripts in `scripts/run` for quick reference.
 
 We evaluated the training performance on MindSpore and Ascend NPUs. The results are as follows.
 
-| Model       | Context      | Precision | BS | NPUs | Resolution | Train T. (s/step) |
-|:------------|:-------------|:----------|:--:|:----:|:----------:|:-----------------:|
-| STDiT2-XL/2 | D910\*-MS2.3 | BF16      | 1  |  1   | 16x512x512 |       2.80        |
+| Model       | Context      | Backend | Precision | BS | NPUs | Resolution(framesxHxW) | Train T. (s/step) |
```
Comment (CaitinZhao) [inline examples/opensora_hpcai/README.md:496]: jit_level ： O1

Comment (Songyuanwei) [reply examples/opensora_hpcai/README.md]: done

File: examples/opensora_hpcai/scripts/infer_t5.py (hunk 1 above)
Comment (Songyuanwei) [reply examples/opensora_hpcai/scripts/infer_t5.py]: done

**Review by CaitinZhao — APPROVED** [review]

**Pull request merged**